integr8ly/ci-cd
# Add plugin install role

## What
Add role for installing plugins required by Delorean jobs.

- [x] Add role for installing plugins
- [x] Add task to the `configure_jenkins` file to install plugins
- [x] Create `plugins.txt` with a list of plugins to be installed

## Verification
1. Run the script
2. Ensure all plugins are installed.
